src/components/checkbox/__control/checkbox__control.bemhtml

```diff
@@ -0,0 +1,16 @@
+block('checkbox').elem('control')(
+    tag()('input'),
+
+    attrs()(function() {
+        const originalAttrs = applyNext();
+
+        originalAttrs.autoComplete = originalAttrs.autocomplete;
```

Comment:
Maybe we can solve this problem here https://github.com/bem-contrib/react-xjst/issues/6